docs/JLBP-0022.md

```diff
@@ -0,0 +1,56 @@
+---
+jlbp:
+  id: JLBP-22
+permalink: /JLBP-22
+---
+
+# Declare all dependencies
```

Comment:
Should this refer explicitly to _direct_ dependencies somehow?

Reply:
yes, done